clio-client/src/main/scala/org/broadinstitute/clio/client/webclient/ClioWebClient.scala

```diff
@@ -54,6 +58,17 @@ object ClioWebClient {
       extends RuntimeException(
         s"Got an error from the Clio server. Status code: $statusCode. Entity: $entityBody"
       )
+
+  def jsonWithDocumentStatus(input: Json): Json = {
+    import ModelAutoDerivation.encodeEnum
+    input.mapObject(
+      obj =>
+        obj.add(
+          "document_status",
```

Comment:
We use this in enough places, it's probably worth making a constant for (maybe in the `Metadata` companion object?)